timeseries-aggregator/src/main/scala/com/expedia/www/haystack/trends/kstream/serde/metric/MetricSerde.scala

```diff
@@ -0,0 +1,11 @@
+package com.expedia.www.haystack.trends.kstream.serde.metric
```

Comment:
Do we want to add new things is a reorg commit?

Reply:
this was there, its just a reorg